.github/workflows/push-tests.yml

```diff
@@ -41,7 +41,10 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-pip-
     - name: Install dependencies
-      run: pip install -r requirements.txt
+      run: |
+        pip install -r requirements.txt
+        brew install moreutils
```

Comment:
Just curious what's this dep for?

Reply:
This is to run the `sponge` which is used by make package for the chalice api-server app.